pkg/prometheus/statefulset.go

```diff
@@ -21,7 +21,7 @@ import (
 	"strings"
 
 	appsv1 "k8s.io/api/apps/v1beta2"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
```suggestion
	"k8s.io/api/core/v1"
```

Reply:
Not sure why vscode keeps trying to make this change when go fmt does not, reverted!